pkg/identityserver/user_registry.go

```diff
@@ -77,11 +78,11 @@ func (is *IdentityServer) validatePasswordStrength(ctx context.Context, password
 	var uppercase, digits, special int
 	for _, r := range password {
 		switch {
-		case r >= 'A' && r <= 'Z':
+		case unicode.IsUpper(r):
```

Comment:
This is quite a significant change to our current definition of uppercase letters. Now it would also include letters like `Д` and `𝒬`, which IMO should be considered "special" in passwords.

Reply:
Why so? Why would we not allow passwords in local languages?
I don't think we should force people to use latin alphabet in passwords, hence I think `Д`, `Ы`, `Ё` etc. are all perfectly valid uppercase characters.